docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,175 @@
+# Autofill Manager
+
+The Tizen autofill framework allows more than one or more Autofill services to be installed on a Tizen device. Autofill manager is a module for managing the installed autofill services. Application developers can use autofill manager to get the current autofill service or change the autofill service.
+
+The main features of the Autofill Manager API include:
+
+- Installed autofill service list
+
+  You can [get the installed autofill service list](#installed-autofill-service-list) in your device.
+
+- Current autofill service
+
+  You can [get the current autofill service](#current-autofill-service).
+
+- Change of autofill service
+
+  You can [set the Autofill service](#change-of-autofill-service).
+
+## Prerequisites
+
+1. To use the Autofill Manager API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__AUTOFILL__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__AUTOFILL__MANAGER__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+   ```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/autofillmanager</privilege>
+   </privileges>
+   ```
+
+2. To use the functions and data types of the Autofill Manager API, include the `<autofill_manager.h>` header file in your application:
+
+    ```c
+    #include <autofill_manager.h>
+    ```
+
+## Initialization
+
+1. To use the Autofill manager library, create an autofill manager handle. It is used in other autofill manager functions as a parameter:
+
```

Comment:
To use the Autofill manager library in your application you must create an autofill manager handle. The Autofill manager handle is used by other Autofill manager functions as a parameter: